localPackages/site_developer/composer.json

```diff
@@ -0,0 +1,40 @@
+{
+    "name": "ttn/site-developer",
+    "description": "Sitepackage for developer training",
```

Comment:
Sitepackage -> Site package

developer training -> TYPO3 Trainer Network trainings

Reply:
I followed https://docs.typo3.org/m/typo3/docs-how-to-document/master/en-us/GeneralConventions/ContentStyleGuide.html#spelling-preferred-terms-reference
where we defined "sitepackage" as a special name within TYPO3:

> sitepackage
    In the context of a TYPO3 sitepackage, a TYPO3 sitepackage extension or the sitepackage builder, we use the spelling sitepackage and not site package.

Should I change that?

Reply:
Okay, then let's keep it that way.